hooks/useGovernanceUnderlyingTokenAccounts.ts

```diff
@@ -0,0 +1,110 @@
+import BigNumber from 'bignumber.js'
+import { useCallback, useEffect, useState } from 'react'
+import { PublicKey } from '@solana/web3.js'
+import { getOwnedTokenAccounts, tryGetMint } from '@utils/tokens'
+import useWalletStore from 'stores/useWalletStore'
+import { SPL_TOKENS } from '@utils/splTokens'
+import { BN } from '@project-serum/anchor'
+import { findATAAddrSync } from '@uxdprotocol/uxd-client'
+
+export type OwnedTokenAccountInfo = {
+  pubkey: PublicKey
+  mint: PublicKey
+  amount: BN
+  uiAmount: number
+  mintDecimals: number
+  mintName: string
+
+  // could be either the Associated Token Account of the mint
+  // or a Token Account
+  isATA: boolean
+}
+
+export type OwnedTokenAccountInfos = OwnedTokenAccountInfo[]
+
+// Loads all the token accounts related to the governance public key
+export default function useGovernanceUnderlyingTokenAccounts(
+  governancePk: PublicKey | null
```

Comment:
Sure need allow it to be `null` ?

Reply:
you are right, we can use undefined instead with `?`